conf/modules.config

```diff
@@ -606,9 +606,6 @@ process{
         ext.prefix       = { meta.num_intervals <= 1 ? "${meta.id}" : "${meta.id}.${target_bed.simpleName}" }
         ext.args         = '--min-alternate-fraction 0.1 --min-mapping-quality 1'
         ext.when         = { params.tools && params.tools.split(',').contains('freebayes') }
-        publishDir       = [
-            enabled: false
-        ]
```

Comment:
Why is this removed?

Reply:
shouldn't be needed, no? default is to not publish

Reply:
@maxulysse did I mix something up? this is ok, right?

Reply:
yes, ok